daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/VersionTimeline.scala

```diff
@@ -55,7 +55,7 @@ private[digitalasset] object VersionTimeline {
       This(That(TransactionVersion("8"))),
       Both(This(ValueVersion("5")), LanguageVersion(LMV.V1, "6")),
       // add new versions above this line (but see more notes below)
-      That(LanguageVersion(LMV.V1, Dev)),
+      Both(This(ValueVersion("6")), LanguageVersion(LMV.V1, Dev)),
```

Comment:
Why not add this as a separate line?

Reply:
I was curious to know if I could do that ?
In particular I was intrigued by the second part of the comment that follow. 

>  do *not* backfill to make more Boths, [...], **except to accompany Dev language versions**;

Reply:
hm, @S11001001 ?

Reply:
Here "add new versions above this line" holds weight; the part you elided [...] is what was being modified by what you bolded, i.e. making Boths with Dev versions doesn't break the timeline. Additionally, Dev versions can appear above the line; that's the only time a backfill alongside a Dev version can hypothetically appear.

Add new versions above this line is the rule here.